zipline/data/loader.py

```diff
@@ -249,6 +249,7 @@ def ensure_benchmark_data(symbol, first_date, last_date, now, trading_day):
         data.to_csv(path)
     except (OSError, IOError, HTTPError):
         logger.exception('failed to cache the new benchmark returns')
+        raise
```

Comment:
@llllllllll I thought the intent here was to catch this failure, warn, and continue?